person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id = rand(1..1000), name = 'Unknown', age = nil, parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements:

![image](https://github.com/SergioPeralta22/MV-School/assets/77069011/5f1fa9ca-7680-4a08-a1ba-fd5618bc0505)

The method `of_age` should be private. Please consider applying this! 🤓 🔧